src/lib/utils/meta_tables/meta_system_utilization_table.cpp

```diff
@@ -0,0 +1,270 @@
+#ifdef __linux__
+#include <numa.h>
+#endif
+
+#ifdef __APPLE__
+#include <mach/mach.h>
+#endif
+
+#include <chrono>
+#include <fstream>
+
+#include "meta_system_utilization_table.hpp"
+
+namespace opossum {
+
+MetaSystemUtilizationTable::MetaSystemUtilizationTable()
+    : AbstractMetaSystemTable(TableColumnDefinitions{{"cpu_system_time", DataType::Long, false},
+                                                     {"cpu_process_time", DataType::Long, false},
+                                                     {"total_time", DataType::Long, false},
+                                                     {"load_average_1_min", DataType::Float, false},
+                                                     {"load_average_5_min", DataType::Float, false},
+                                                     {"load_average_15_min", DataType::Float, false},
+                                                     {"system_memory_free", DataType::Long, false},
+                                                     {"system_memory_available", DataType::Long, false},
+                                                     {"process_virtual_memory", DataType::Long, false},
+                                                     {"process_physical_memory_RSS", DataType::Long, false}}) {}

```

Comment:
What does "physical memory" add here? Wouldn't `process_rss` say the same thing?